tools/src/versioning/ios/versionHermes.ts

```diff
@@ -0,0 +1,240 @@
+import spawnAsync from '@expo/spawn-async';
+import fs from 'fs-extra';
+import os from 'os';
+import path from 'path';
+
+import { EXPO_DIR } from '../../Constants';
+import { GitDirectory } from '../../Git';
+import logger from '../../Logger';
+import { transformFilesAsync } from '../../Transforms';
+import type { FileTransform } from '../../Transforms.types';
+import { searchFilesAsync } from '../../Utils';
+
+const TRANSFORM_HEADERS_API = ['hermes', 'DebuggerAPI'];
+const TRANSFORM_HEADERS_PUBLIC = [
+  'RuntimeConfig',
+  'CrashManager',
+  'CtorConfig',
+  'DebuggerTypes',
+  'GCConfig',
+  'GCTripwireContext',
+];
+
+const VERSIONED_JSI_DIR = 'versioned-jsi';
+const REACT_NATIVE_FORK_DIR = path.join(EXPO_DIR, 'react-native-lab', 'react-native');
```

Comment:
There is `getReactNativeSubmoduleDir` in `Directories.ts` already. Maybe let's add it as a constant to `Constats.ts`, just like `EXPO_DIR`?